gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/views/privacySettings.handlebars

```diff
@@ -46,8 +46,9 @@
                             </button>
 
                             <input type="hidden" class="gpiic-oauth2-privacySettings-serviceName" name="serviceName" value="{{serviceName}}" />
-                            <input type="hidden" class="gpiic-oauth2-privacySettings-authDecisionId" name="authDecisionId" value="{{authDecisionId}}" />
-                            <input type="hidden" class="gpiic-oauth2-privacySettings-oauth2ClientId" name="oauth2ClientId" value="{{oauth2ClientId}}" />
+                            <input type="hidden" class="gpiic-oauth2-privacySettings-authorizationId" name="authorizationId" value="{{authorizationId}}" />
+                            <input type="hidden" class="gpiic-oauth2-privacySettings-clientId" name="clientId" value="{{clientId}}" />
+                            <input type="hidden" class="gpiic-oauth2-privacySettings-solutionId" name="solutionId" value="{{solutionId}}" />
```

Comment:
It looks to me like the `solutionId` field might not be used. I ran the `gpii.config.cloudBased.development.all.local` config locally and logged into `privacy-settings` with user "li". I then added the "Windows Magnifier" service and inspected the DOM. I see the `clientId` as set to `com.microsoft.windows.magnifier` but `solutionId` as empty.